### PR TITLE
Add link to the GoGo Upgrade wiki page

### PR DESCRIPTION
**Pull request by Idloj**

### CODE REVIEW

**Comment by SethTisue** [comment]
review by, I dunno, @cbradyatinquire?

**Comment by mrerrormessage** [comment]
The link just above this (to gogo-rxtx) refers to the version of the gogo extension prior to serial, while the transition guide is on how to go from serial to HID. This addition would be most clear if it appeared at the end of the paragraph ending L62.

**Comment by Idloj** [comment]
Like this?

File: docs/transition.html [hunk 1]
```diff
@@ -61,6 +61,8 @@
       gogo-serial extension.  Change your model to use gogo-serial as opposed to
       gogo.
     <p>
+      More details [here](https://github.com/NetLogo/NetLogo/wiki/GoGo-Upgrade).
```
Comment (Idloj) [inline docs/transition.html]: Like this?

Comment (mrerrormessage) [inline docs/transition.html]: It doesn't need a separate `<p>` tag. It can go just after the prior sentence ends.

**Comment by mrerrormessage** [comment]
Looks good, thanks for this change!

**Pull request merged**